freqtrade/commands/cli_options.py

```diff
@@ -262,6 +262,14 @@ def __init__(self, *args, **kwargs):
         metavar="INT",
         default=constants.HYPEROPT_EPOCH,
     ),
+    "early_stop": Arg(
+        "-es",
```

Comment:
I'd prefer not to have 2 letter "shortcuts" (they're no longer short - and can cause pretty much ambiguity, too).
do we really need a shortcut here, or is `--early-stop` more expressive?

Reply:
ok

Reply:
fixed docs for CI. 
the test can be small so it can finish fast. something like a hyperopt with 10 epochs with early stop of 3.
The idea is that it will stop hyperopt after x epochs with no improvement in loss, not when space is exhausted.

Reply:
Doesn't seem to work, then ... 
i did a test (limited to sell space) - and got seemingly the same result for 20 epochs.
`--early-stop 3` didn't stop it - but it continued through through all 20 epochs

I'd expect this to stop at latest at epoch 9 or 10 (i used -j 3 - so parallelism will have it "finish" that batch).

it did load it correctly based on the logs 

```
2025-05-17 09:35:48,781 - freqtrade.configuration.configuration - INFO - Parameter --early-stop detected ... Will early stop hyperopt if no improvement after 3 epochs ...  
```

but didn't stop.
![image](https://github.com/user-attachments/assets/d6b072f1-8d89-44e4-8f6d-4e260c992eea)

Reply:
Setting a higher number of epochs (100 let's say) will work.
<img width="800" alt="Screenshot 2025-05-17 at 11 38 00" src="https://github.com/user-attachments/assets/06ee67a3-ee74-4d0b-8333-99de6263c48b" />
Do you have the latest version? I fixed something yesterday

Reply:
Yes i do, and yes, with more epochs it works :+1: